.github/workflows/deploy.yaml

```diff
@@ -3,7 +3,8 @@ name: Deploy to Heroku
 on:
   push:
     branches:
-      - main
+      # - main
+      - "**" # Matches all branch and tag names.
```

Comment:
I actually worked with George on this and he said lines 5,6,7 can all be commented! This can make it easier but if not its also fine the way it is ^_^